finality-aleph/src/block/substrate/verification/cache.rs

```diff
@@ -298,8 +307,8 @@ where
             sp_timestamp::Timestamp::current(),
             sp_consensus_slots::SlotDuration::from_millis(MILLISECS_PER_BLOCK),
         );
-        if *slot > slot_now + HEADER_VERIFICATION_SLOT_OFFSET {
-            return Err(VerificationError::HeaderVerification(HeaderTooNew(*slot)));
+        if slot.1 > slot_now + HEADER_VERIFICATION_SLOT_OFFSET {
+            return Err(VerificationError::HeaderVerification(HeaderTooNew(slot.1)));
```

Comment:
Eh, this would require annoying changes in other files and carries little relevant information – it's the slot that's too new, regardless of the session.